main.cpp

```diff
@@ -134,30 +203,32 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 int main()
 {
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
+    auto carRented = rentACar (6, 2); 
     //1)
-    
+    auto reverb = reverbParameters (7, 20);
     //2)
-    
+    auto compress = compressorParameters (100, 0);
     //3)
-    
+    auto delay = delayTime (21, 33);
     //4)
-    
+    auto chorus = chorusEffect (21.3f, 44.2f);
     //5)
-    
+    auto mod = modulation (21.1, 45.3);
     //6)
-    
+    auto dist = distortion (1, 0);
     //7)
-    
+    auto FX = effectOn (0,1);
     //8)
-    
+    auto signal = processSignal (23.2, 78.2);
     //9)
-    
+    auto msg = message (33.3f);
     //10)
+    auto answer =returnAnswer (1);

```

Comment:
Fix the formatting here.   Your work needs to conform to the course coding standard
- no spaces around `(` and `)`
- spaces around `=` 
- spaces after `,` 
You need to train your eye to see code that is formatted differently from the coding standard being used in whatever project or team you're working with.
- [ ] revise